docs/basic_training/channels.md

```diff
@@ -693,64 +693,162 @@ Files containing JSON content can also be parsed:
 
 This can also be used as a way to parse YAML files:
 
-```groovy linenums="1"
-import org.yaml.snakeyaml.Yaml
+=== "Source code"
 
-def f = file('data/meta/regions.yml')
-def records = new Yaml().load(f)
+    ```groovy linenums="1"
+    import org.yaml.snakeyaml.Yaml
 
+    def f = file('data/meta/regions.yml')
+    def records = new Yaml().load(f)
 
-for (def entry : records) {
-    log.info "$entry.patient_id -- $entry.feature"
-}
-```
+
+    for (def entry : records) {
+        log.info "$entry.patient_id -- $entry.feature"
+    }
+    ```
+
+=== "data/meta/regions.yml"
+
+    ```yml
```

Comment:
Syntax highlighting isn't working here. Could be that the indentation isn't right and it's breaking it? Or could be something silly like it needing `yaml` instead of `yml`..

````suggestion
    ```yaml
````